SDWebImage/UIImage+GIF.m

```diff
@@ -87,7 +87,8 @@ + (UIImage *)sd_animatedGIFNamed:(NSString *)name {
     CGFloat scale = [UIScreen mainScreen].scale;
 
     if (scale > 1.0f) {
-        NSString *retinaPath = [[NSBundle mainBundle] pathForResource:[name stringByAppendingString:@"@2x"] ofType:@"gif"];
+        int imgScale = [NSNumber numberWithFloat:scale].intValue;
+        NSString *retinaPath = [[NSBundle mainBundle] pathForResource:[name stringByAppendingString:[NSString stringWithFormat:@"@%ix",imgScale]] ofType:@"gif"];
```

Comment:
please put `[NSString stringWithFormat:@"@%ix",imgScale]` into a separate variable for readability